src/extension/background-script/connectors/commando.ts

```diff
@@ -0,0 +1,151 @@
+import LnMessage from "lnmessage";
+import { v4 as uuidv4 } from "uuid";
+
+import Connector, {
+  CheckPaymentArgs,
+  CheckPaymentResponse,
+  GetBalanceResponse,
+  GetInfoResponse,
+  GetInvoicesResponse,
+  KeysendArgs,
+  MakeInvoiceArgs,
+  MakeInvoiceResponse,
+  SendPaymentArgs,
+  SendPaymentResponse,
+  SignMessageArgs,
+  SignMessageResponse,
+} from "./connector.interface";
+
+interface Config {
+  host: string;
+  port: number;
+  rune: string;
+  pubkey: string;
+  wsProxy: string;
+  privateKey: string;
+}
+
+type CommandoGetInfoResponse = {
+  alias: string;
+  id: string;
+  color: string;
+};
+type CommandoMakeInvoiceResponse = {
+  bolt11: string;
+  payment_hash: string;
+  payment_secret: string;
+};
+type CommandoChannel = {
+  peer_id: string;
+  channel_sat: number;
+  amount_msat: number;
+  funding_txid: string;
+  funding_output: number;
+  connected: boolean;
+  state: string;
+};
+type CommandoListFundsResponse = {
+  channels: CommandoChannel[];
+};
+
+export default class Commando implements Connector {
+  config: Config;
+  ln: LnMessage;
+
+  constructor(config: Config) {
+    this.config = config;
+    this.ln = new LnMessage({
+      remoteNodePublicKey: this.config.pubkey,
+      wsProxy: this.config.wsProxy || "wss://lnwsproxy.regtest.getalby.com",
+      ip: this.config.host,
+      port: this.config.port || 9735,
+      privateKey:
+        this.config.privateKey ||
+        "d6a2eba36168cc31e97396a781a4dd46dd3648c001d3f4fde221d256e41715ea",
+    });
+  }
+
+  async init() {
+    // initiate the connection to the remote node
+    await this.ln.connect();
```

Comment:
as this is a promise I'd remove the async and await and simple do return `this.ln.connect()`

Reply:
![Schermafbeelding 2022-10-18 om 09 29 24](https://user-images.githubusercontent.com/33457577/196365070-f57658de-e005-4c17-9538-2d07292801f0.png)

Reply:
ahhhh. I see. do we need to check the response there? 
```
const connected =  await this.ln.connect();
if (!connected) { 
   console.error("Failed to connect");
}
```

(currently we assume the connection works (init() does not have any error handling - which it should have)

Reply:
I don't think we need to check the response